File: csharp/DVSClient/DVSClient/server/address/layout/RestApiGetLayoutLayout.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Layout
{
    public class RestApiGetLayoutLayout
    {
        [JsonProperty("id")]
        public string? Id { get; set; }

        [JsonProperty("name")]
        public string? Name { get; set; }

        [JsonProperty("comment")]
        public string? Comment { get; set; }

        [JsonProperty("applies_to")]
        public IEnumerable<RestApiAddressLayoutAppliesTo>? AppliesTo { get; set; }

        [JsonProperty("lines")]
        public IEnumerable<RestApiAddressLayoutLine>? Lines { get; set; }

        [JsonProperty("status")]
        public string? Status { get; set; }

        [JsonProperty("license_id")]
        public string? LicenseId { get; set; }
    }
}

File: typescript/src/server/address/layout/restApiGetLayoutLayout.ts
import { RestApiAddressLayoutAppliesTo } from "./restApiAddressLayoutAppliesTo";
import { RestApiAddressLayoutLine } from "./restApiAddressLayoutLine";

export type RestApiGetLayoutLayout = {
    id?: string;
    name?: string;
    comment?: string;
    applies_to?: RestApiAddressLayoutAppliesTo[];
    lines?: RestApiAddressLayoutLine[];
    status?: string;
    license_id?: string;
};